sources/packages/api/scripts/setupDB.ts

```diff
@@ -1,29 +1,23 @@
-require("../env_setup");
-import { createConnection } from "typeorm";
-const config = require("../ormconfig");
+import { ormConfig, simsDataSource } from "../src/database/config/ormconfig";
```

Comment:
Typeorm says as per their OOTB CLI setup, to declare the data source in data-source.ts.
![image](https://user-images.githubusercontent.com/54600590/180046540-aea13d2f-d7a9-4a1f-ac18-b91de400aca9.png)